frontend/components/YamlAce/_styles.scss

```diff
@@ -1,20 +1,4 @@
 .yaml-ace {
-  &__label {
-    font-size: $small;
-    font-weight: $bold;
-    font-style: normal;
-    font-stretch: normal;
-    letter-spacing: -0.5px;
-    color: $core-fleet-black;
-    display: block;
-    margin: 0 0 $pad-xsmall;
-    min-height: 25px;
-
-    &--error {
```

Comment:
This color needs to be added back into `yaml-ace__label--error`

<img width="1258" alt="Screenshot 2024-01-17 at 9 55 39 AM" src="https://github.com/fleetdm/fleet/assets/71795832/32f79476-4614-48da-9782-c394d4a8e2bf">

Reply:
Good catch, thanks @RachelElysia. This is a great case study in moving from how we were handling styling (locally) to what we want now, which is keeping everything in line with the _global_ styles as of this PR. So, my miss here was that the `--error` suffix is still being appended to the local `yaml-ace__label`, but not the global `form-field__label`, which would have the correct styling applied. I'll add that in. 
![Screenshot 2024-01-17 at 10 44 48 AM](https://github.com/fleetdm/fleet/assets/61553566/3b3a54b5-6f21-4c52-8a09-52f616b2b3ec)